static/styles/portico.css

```diff
@@ -1169,7 +1169,7 @@ input#zulipbot:checked ~ #tab-zulipbot {
 .login-form {
     margin: auto;
     /* plus input padding. */
-    width: calc(300px + 28px);
+    width: calc(300px - -28px);
```

Comment:
This one makes sense to me to fix
gif attached.
![calc-2](https://user-images.githubusercontent.com/17237412/36866857-833b87b2-1db9-11e8-9b67-83099b8f9c33.gif)